Paws4Me/Paws4Me/Extensions/UIColorExtension.swift

```diff
@@ -10,12 +10,12 @@ import UIKit
 
 extension UIColor {
     static var myAppPurple: UIColor {
-        return UIColor(named: "primaryPurple") ?? UIColor.clear
+        return UIColor(named: "primaryPurple")!
```

Comment:
So I see you and Lloyd had a conversation about this, so happy to go with this 😄 but please be watchful with something like this as your app would crash if someone changes the name in the assets folder -which is unlikely when joining an existing project but if you were to start a project from scratch it would be very possible.

Reply:
Thank you. What is the convention in industry? Do you set a default colour or force unwrap?